inc/Engine/Media/AboveTheFold/Frontend/Controller.php

```diff
@@ -146,9 +146,17 @@ private function set_fetchpriority( $lcp, string $html ): string {
 			return $html;
 		}
 
-		$url  = preg_quote( $lcp->src, '/' );
+		$url     = preg_quote( $lcp->src, '/' );
+		$pattern = '#<img(?:[^>]*?\s+)?src=["\']' . $url . '["\'](?:\s+[^>]*?)?>#';
+		if ( wp_http_validate_url( $lcp->src ) && ! $this->is_external_file( $lcp->src ) ) {
+			$url = get_rocket_parse_url( $lcp->src );
+			$url = preg_quote( $url['path'], '/' );
```

Comment:
why do we need to use preg_quote again? we already used it above with the whole url and I believe this is enough, what do u think?

Reply:
The `$url` above is overwritten, preg_quote is use here for the path